cmd/bosun/sched/template.go

```diff
@@ -189,7 +189,7 @@ func (c *Context) eval(v interface{}, filter bool, series bool, autods int) (exp
 		return nil, "", fmt.Errorf("%v: %v", v, err)
 	}
 	if filter {
-		e, err = expr.New(opentsdb.ReplaceTags(e.String(), c.State.Group))
+		e, err = expr.New(opentsdb.ReplaceTags(e.Text, c.State.Group))
```

Comment:
This commit is totally unrelated to the rest. Could you please submit it as a separate PR? And remove it from this PR. I'd appreciate squashing all these commits together into a single commit, too.